### DISCUSSION TITLE
How to ignore HttpServletResponse and HttpServletRequest parameters

### DISCUSSION THREAD

**Discussion by mu-qiao** [TongchengOpenSource/smart-doc#637] [Q&A] [answered]

**Comment by mu-qiao** [comment]
If your method parameters include HttpServletResponse and HttpServletRequest, and you wish to hide these two parameters, what should you do?
方法参数有HttpServletResponse，HttpServletRequest 参数，我想屏蔽这个两个参数应该怎么做

**Comment by netdied** [comment, marked as answer]
Which version are you using? These two parameters are automatically shielded.
用的哪个版本？这两个参数会自动屏蔽的
![image](https://github.com/TongchengOpenSource/smart-doc/assets/20217389/88a74917-f81f-4676-856e-9870458d4f11)

**Comment by mu-qiao** [comment]
2.4.2
![image](https://github.com/TongchengOpenSource/smart-doc/assets/80091862/2f6a13c4-2930-45b1-983e-30b453eda949)
![image](https://github.com/TongchengOpenSource/smart-doc/assets/80091862/6cbc80e2-a4ab-4740-81c8-7b54cc80fa90)

**Comment by mu-qiao** [comment]
Using version 2.7.7 and later resolves this issue, thank you.
使用2.7.7之后这个问题解决了，感谢
